omnibus/files/private-chef-ctl-commands/secrets.rb

```diff
@@ -27,13 +29,43 @@
   "opscode-reporting" => ["rabbitmq_password", "sql_password", "sql_ro_password"],
 }
 
+SERVICES_REQUIRING_RESTART = {
```

Comment:
We considered this but thought it was not worth the hassle vs what we ended up with.